File: cpp/10.cpp
#include <iostream>
#include <math.h> 
using namespace std;

bool ft_isPrime(int n)
{
	int i;

	for (i = 2; i < int(sqrt(n) + 1); i++)
	{
		if (n % i == 0)
			return (false);
	}
	return (true);
}

int main()
{
	int i;
	long int sum;

	sum = 0;
	for (i = 2; i < 2000000; i++)
		sum += ft_isPrime(i) ? i : 0;
	cout << sum << endl;

	return(0);
}

File: Python3/10/10.py
#!/usr/bin/python3

def ft_isPrime(n):
	for i in range(2,int(n**0.5 +1)):
		if n % i == 0:
			return (False)
	return (True)

result = sum([i for i in range(2,2_000_000) if ft_isPrime(i)])
print(result)